fiftyone/utils/unlimited_ocr.py

```
@@ -0,0 +1,255 @@
+
+        if coords and isinstance(coords[0], (int, float)):
+            coords = [coords]
+
+        for box in coords:
+            if len(box) != 4:
+                continue
+
+            x1, y1, x2, y2 = (float(v) / (_NUM_COORD_BINS - 1) for v in box)
+            x0 = min(max(min(x1, x2), 0.0), 1.0)
+            y0 = min(max(min(y1, y2), 0.0), 1.0)
+            xf = min(max(max(x1, x2), 0.0), 1.0)
```

Ruff (0.15.21)

[warning] Nested `max` calls can be flattened

Flatten nested `max` calls

(PLW3301)